### PR TITLE
Shared Crumb Style Updates

### PR DESCRIPTION
**Pull request by cameron-freshworks**
*Issue #:* /bcgov/entity#10233

*Description of changes:*
* Updated class binding to explicitly set cursor.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).

### CODE REVIEW

File: src/components/BreadCrumb/BreadCrumb.vue [hunk 1]
```diff
@@ -107,6 +112,10 @@ export default class BreadCrumb extends Vue {
   cursor: pointer !important;
 }
 
+.inactive-crumb {
+  cursor: default !important;
+}
```
Comment (severinbeauvais) [inline src/components/BreadCrumb/BreadCrumb.vue:117]: 1. I think `!important` can still be overridden by [more specific CSS](https://www.w3schools.com/css/css_specificity.asp). In some cases I have had to use cascading selectors to override something ([example](https://github.com/severinbeauvais/business-filings-ui/blob/8991a1fdabdb37e57bc6205b020d10d1809147df/src/components/dialogs/StaffPaymentDialog.vue#L106)).

2. Would [pointer events](https://www.w3schools.com/cssref/css3_pr_pointer-events.asp) be another, or alternate, property that meets your need?

File: src/interfaces/breadcrumb-interface.ts [hunk 2]
```diff
@@ -2,4 +2,5 @@ export interface BreadcrumbIF {
   text: string
   to?: any
   href?: string
+  disabled?: boolean
```
Comment (severinbeauvais) [inline src/interfaces/breadcrumb-interface.ts:5]: Obsolete?

Comment (cameron-freshworks) [reply src/interfaces/breadcrumb-interface.ts:5]: removed in next commit

File: src/components/BreadCrumb/BreadCrumb.vue (hunk 1 above)
Comment (cameron-freshworks) [reply src/components/BreadCrumb/BreadCrumb.vue:117]: 1: In this instance it's working as expected. 
2: I believe it would do the same thing yes.

Comment (severinbeauvais) [reply src/components/BreadCrumb/BreadCrumb.vue:117]: OK, perfect.

File: src/components/BreadCrumb/BreadCrumb.vue [hunk 3]
```diff
@@ -107,6 +112,10 @@ export default class BreadCrumb extends Vue {
   cursor: pointer !important;
 }
 
+.inactive-crumb {
+  cursor: default !important;
```
Comment (severinbeauvais) [inline src/components/BreadCrumb/BreadCrumb.vue:116]: Is it worth a comment to describe why this is needed?

**Review by severinbeauvais — APPROVED** [review]

**Pull request merged**